### DISCUSSION TITLE
why image color changed after composite

### DISCUSSION THREAD

**Discussion by Matiz7** [ImageMagick/ImageMagick#5928] [Digital Image Processing] [answered]
https://focusmedia-dmp-sandbox.oss-cn-shanghai.aliyuncs.com/test/425575_920.jpg
https://focusmedia-dmp-sandbox.oss-cn-shanghai.aliyuncs.com/test/电梯海报425_575.jpg
https://focusmedia-dmp-sandbox.oss-cn-shanghai.aliyuncs.com/test/425575_1.jpg

composite -geometry +426+490 425575_920.jpg 电梯海报425_575.jpg 425575_1.jpg

425575_1.jpg's color obviously diff with 425575_920.jpg

**Comment by snibgo** [comment, marked as answer]
```
C:\web\im>%img7%magick 425575_920.jpg info:
425575_920.jpg JPEG 920x1244 920x1244+0+0 8-bit CMYK 916851B 0.000u 0:00.000

C:\web\im>%img7%magick 电梯海报425_575.jpg info:
ç"µæ¢¯æµ·æS¥425_575.jpg JPEG 1836x2349 1836x2349+0+0 8-bit sRGB 453387B 0.000u 0:00.000
```
This shows us that one input is sRGB and the other is CMYK. You should first make them both encoded in the same colorspace.
